src/api/app/models/staging_workflow.rb

```diff
@@ -32,6 +32,33 @@ def ignored_requests
     BsRequest.none # TODO: define this method
   end
 
+  def to_xml
+    builder = Nokogiri::XML::Builder.new
+    staging_workflow_attributes = { project: project.name,
+                                    who: User.current_login,
+                                    when: updated_at }
+
+    builder.staging_workflow(staging_workflow_attributes) do |builder_staging_workflow|
+      staging_projects.each do |staging_project|
+        builder_staging_workflow.staging_project(name: staging_project.name) do |builder_staging_project|
+          staging_project.staged_requests.each do |staged_request|
+            request_attributes = { number: staged_request.number,
```

Comment:
I don't think the staged requests belong in the backend. this would be *super* noisy on _history

Reply:
But that's the only thing that changes...

Reply:
well, in daily routine yes - but it will be basically impossible to find any other changes in Factory's _history.

So storing the selection history within the staging project might make sense - but I would limit _staging_workflow to changes to the *workflow*, not the work done within

Reply:
Okay, sounds reasonable to me. @DavidKang please remote requests.